File: Day Three/Java/RandomArithmetic.java
import java.util.Scanner;
import java.util.Random;


public class RandomArithmetic{

	public static void main(String...args){
		Scanner input = new Scanner(System.in);


		// System.out.println(random.nextBoolean());

		for(int count = 1; count <=10; count++){
			
			Random random = new Random();
			int digitOne = random.nextInt(20)+ 1;
			int digitTwo = random.nextInt(10)+1;

			Random operatorChoice = new Random();
			int operator = operatorChoice.nextInt(4));

			String operatorSwitch = """
			int bad
			int good
			""";

			switch(operator){

				case 0: {
					operatorSwitch = "+";
					good = digitOne + digitTwo;
					break;
				}
				case 1: {
					operatorSwitch = "-";
					good = digitOne - digitTwo;
					break;
				}
				case 2: {
					operatorSwitch = "*";
					good = digitOne * digitTwo;
					break;
				}
				case 3: {
					operatorSwitch = "/";
					good = digitOne * digitTwo;
					break;
				}
				default:{
					System.out.println("Invalid input");
				}

			}
			//System.out.pr
		}

	}



}

File: Day Three/Python/random_arithmetic.py
import random

correct_answer_count = 0
wrong_answer_count = 0

for _ in range(10):
	digit_one = random.randint(1,20)
	digit_two = random.randint(1,10)

	if digit_one <= 5:
		random_arithmetic = digit_one + digit_two
		print(digit_one, " + ", digit_two)
		user_response = int(input("Enter the answer: "))
		if random_arithmetic == user_response:
			print("Correct !!")
			correct_answer_count +=1
		else:
			print("Wrong answer")
			wrong_answer_count += 1

	elif digit_one >5 and digit_one <= 10 :
		random_arithmetic = digit_one - digit_two
		print(digit_one, " - ", digit_two)
		user_response = int(input("Enter the answer: "))
		if random_arithmetic == user_response:
			print("Correct !!")
			correct_answer_count +=1
		else:
			print("Wrong answer")
			wrong_answer_count += 1

	elif digit_one >10 and digit_one <= 15 :
		random_arithmetic = digit_one * digit_two
		print(digit_one, " X ", digit_two)
		user_response = int(input("Enter the answer: "))
		if random_arithmetic == user_response:
			print("Correct !!")
			correct_answer_count +=1
		else:
			print("Wrong answer")
			wrong_answer_count += 1

	else:
		random_arithmetic = digit_one / digit_two
		print(digit_one, " / ", digit_two)
		user_response = int(input("Enter the answer: "))
		if random_arithmetic == user_response:
			print("Correct !!")
			correct_answer_count +=1
		else:
			print("Wrong answer")
			wrong_answer_count += 1

print("Correct answers = ", correct_answer_count)
print("Incorrect answers = ", wrong_answer_count)
